From 4b43402ccbe59cc4804964ff7128ea13353701ad Mon Sep 17 00:00:00 2001
From: Dmitry Petrochenko <dmitry.petrochenko@intel.com>
Date: Thu, 18 Apr 2013 15:10:21 +0700
Subject: Dalvik: Remove "VRDELAY_CAN_THROW was not optimized" message

BZ: 101609

The message itself does not indicate any error, it just a hint for further
optimization of VRDELAY_CAN_THROW case. This message increase number of bugs
initially triaged as Dalvik issues (incorrectly, just because of message).
thus, we remove this message.

Category: device-enablement
Domain: AOSP-Dalvik-Compiler-CG
Origin: internal
Upstream-Candidate: no, needs rework

Change-Id: I81af42ddb282aa3394fa9a9705291c681e091301
Orig-MCG-Change-Id: I13183a3b577f82c111af5dac96eab44a30c9b496
Signed-off-by: Dmitry Petrochenko <dmitry.petrochenko@intel.com>
Reviewed-on: http://android.intel.com:8080/128462
Reviewed-by: Lupusoru, Razvan A <razvan.a.lupusoru@intel.com>
Reviewed-by: Beyler, Jean Christophe <jean.christophe.beyler@intel.com>
Reviewed-by: Katkov, Serguei I <serguei.i.katkov@intel.com>
Reviewed-by: lab_aqa <lab_aqa@intel.com>
Reviewed-by: Semukhina, Elena V <elena.v.semukhina@intel.com>
Reviewed-by: Chen, Dong-Yuan <dong-yuan.chen@intel.com>
Tested-by: Chen, Dong-Yuan <dong-yuan.chen@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
Signed-off-by: Serguei Katkov <serguei.i.katkov@intel.com>
---
 vm/compiler/codegen/x86/lightcg/Lower.cpp |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/vm/compiler/codegen/x86/lightcg/Lower.cpp b/vm/compiler/codegen/x86/lightcg/Lower.cpp
index 0d19163..f99ea0f 100644
--- a/vm/compiler/codegen/x86/lightcg/Lower.cpp
+++ b/vm/compiler/codegen/x86/lightcg/Lower.cpp
@@ -589,8 +589,6 @@ int lowerByteCodeCanThrowCheck(const Method* method, const MIR * mir, const u2 *
         bool state_changed = cancelVRFreeDelayRequestAll(VRDELAY_CAN_THROW);
         if(state_changed==true) {
             // Not optimized case (delay was not canceled inside bytecode generation)
-            ALOGI("JIT_INFO: VRDELAY_CAN_THROW cancel was not optimized for bytecode=0x%x",
-                  mir->dalvikInsn.opcode);
             // Release all remaining VRDELAY_CAN_THROW requests
             do {
                 state_changed = cancelVRFreeDelayRequestAll(VRDELAY_CAN_THROW);
-- 
1.7.4.1

